cmd/gateway/s3x/s3.proto

```diff
@@ -28,29 +28,20 @@ message InfoResponse {
 // for keeping track of buckets, objects, and their corresponding IPFS hashes
 message Ledger {
     // key = bucket name
-    map<string, LedgerBucketEntry> buckets = 1 [(gogoproto.nullable) = false];
+    map<string, LedgerBucketEntry> buckets = 1;
     // key = partID
-    map<string, MultipartUpload> multipartUploads = 2 [(gogoproto.nullable) = false];
```

Comment:
We should keep the nullable option since it removes the pointer and will allow us to unmarshal/marshal the data faster

Reply:
I thought more about this, I'm converting this away from saving as a single object for efficiency, and because there this no way to get a safe pointer from a map value, using a pointer is more convenient to code.

Reply:
Sounds good